javascript.md

```diff
@@ -71,18 +94,37 @@ Object.keys(person).forEach(lang.hitch(this, function(prop){
 });
 ```
 
+## Tools and configuration
+
+### Generic
+
+- [Jshint file][jshint-file]
+
+### Sublime Text
+
+- Package Control [installation][package-control-install]
+- Basic [Sublime Text User settings][sublime-settings]
+- Sublime Text [Package Control settings][sublime-package-control] with some nice packages to have.
+
 ### ChangeLog
 
 #### 2015.05.08
 
 - Initial Draft
 
+- Add more general tips.
```

Comment:
I'd suggest adding a new date above the old one and a little more detail. Something like:

```markdown
#### 2015.05.13

- More general tips
- Add performance gotchas
- Add tools & settings
```